src/components/molecules/HeaderDesktop/HeaderDesktop.tsx

```diff
@@ -0,0 +1,10 @@
+const HeaderDesktop = () => (
```

Comment:
Why not just `Header` ?

Reply:
Because there will be also a separate component for HeaderMobile